tests/x11/rstudio_prepare.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Install the rstudio base package and Firefox
+# Maintainer: Dan Čermák <dcermak@suse.com>
+
+use base "x11test";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run() {
+    # increase timeout to 15 mins, shouldn't take as long, but who knows...
+    # FIXME: remove git
```

Comment:
so what about the FIXME?

Reply:
I need to submit a fix to rstudio to be able to remove this.

Reply:
ok then please avoid the word "FIXME" here. They tend to stay in code forever :)

Reply:
So should I remove it and make a TODO note for myself?

Reply:
yes, please. And just in case you never come to it the test will still be valid and work :)

Reply:
And it's gone!